examples/astro/src/pages/codehike.mdx

```diff
@@ -0,0 +1,14 @@
+import "@code-hike/mdx/styles.css"
+import { CH } from "@code-hike/mdx/components"
```

Comment:
Specifying an explicit import for `CH` allows Astro to properly hydrate the component (see comment about disabling `autoImport` for additional context)